client/app/core/constants.js

```diff
@@ -4,6 +4,8 @@
 
   angular.module('app.core')
     .constant('lodash', _)
+    // eslint-disable-next-line
```

Comment:
This one is not like the others :)

Looking at the whole file, we should be consistent - so no `window`, and no comment, but add it to the list on the first line :)